### ISSUE TITLE
[2.0.0-beta.9] Custom rules defined using Submission::EVENT_DEFINE_RULES are not working

### ISSUE THREAD

**Issue by mtnorthrop** [verbb/formie#903]
**Description**
Custom validation rules that are defined using the `Submission::EVENT_DEFINE_RULES` event aren't included in the array returned by `Submission::defineRules()`.

It appears that the problem is at the end of the `Submission::defineRules()` method here:
https://github.com/verbb/formie/blob/cfa4e381403ce0a27d392c718f8027dac43e45ea/src/elements/Submission.php#L1032-L1039

In 2.0.0-beta.9, the original `$rules` array is being returned, instead of `$event->rules`. I see that in 1.5.17, `$event->rules` is what's being returned:
https://github.com/verbb/formie/blob/ddd39d2152ca4b29968e3ae1365e7582945e23b1/src/elements/Submission.php#L269-L292

**Additional info**
- Plugin version: 2.0.0-beta.9
- Craft version: 4.0.3

**Comment by engram-design** [comment]
Thanks for the heads-up, fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.0-beta.9",
  "...": "..."
}
```

Then run `composer update`.

**Comment by mtnorthrop** [comment]
Works perfectly, thanks!

**Comment by engram-design** [comment]
Fixed in [2.0.0-beta.10](https://github.com/verbb/formie/releases/tag/2.0.0-beta.10)
